src/validations/chat/chatInputValidations.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from 'zod';
+
+export const chatInputValidations = z.object({
+	message: z.string().nonempty({ message: 'Message required! :)' }),
+});
```

Comment:
Let's avoid user to long message in out chat by adding min and max validations this message, you limiy ot 255 characters.

Reply:
>I have set min characters to be 1 and max characters to be 255. we don't console any message